OpenRA.Mods.RA/Traits/Infiltration/InfiltrateForActor.cs

```diff
@@ -13,18 +13,18 @@
 
 namespace OpenRA.Mods.RA.Traits
 {
-	class InfiltrateForSupportPowerInfo : ITraitInfo
+	class InfiltrateForActorInfo : ITraitInfo
```

Comment:
You could also undo the `InfiltrateForSupportPower` → `InfiltrateForActor` renaming and leave that for another pull request. It doesn't seem a requirement here, but more like a "oh, while you are at it..." which is probably not the best thing to ask new contributors.